app/router.js

```diff
@@ -154,4 +154,8 @@ Router.map(function () {
     path: 'data/programyears/:program_year_id/objectives',
   });
   this.route('search');
+  this.route('reports', function () {
+    this.route('subject', { path: '/reports/subject/:report_id' });
```

Comment:
```suggestion
    this.route('subject', { path: '/reports/subjects/:report_id' });
```
keeping plural URL to match everything else.

Reply:
:heavy_check_mark: